docs/containers-transports.5.md

```diff
@@ -0,0 +1,102 @@
+
+An image located in a local containers storage.
+The format of _docker-reference_ is described in detail in the **docker** transport.
+
+The _storage-reference_ allows for referencing storage locations on the file system and has the format `"[driver@root]"` where driver refers to the storage driver (e.g., overlay or btrfs) and where root is an absolute or a relative path to the storage's root directory.
+Please refer to containers-storage.conf(5) for further information.
+
+### **dir:**_path_
+
+An existing local directory _path_ storing the manifest, layer tarballs and signatures as individual files.
+This is a non-standardized format, primarily useful for debugging or noninvasive container inspection.
+
+### **docker://**_docker-reference_
+
+An image in a registry implementing the "Docker Registry HTTP API V2".
+By default, uses the authorization state in `$XDG_RUNTIME_DIR/containers/auth.json`, which is set using podman-login(1).
+If the authorization state is not found there, `$HOME/.docker/config.json` is checked, which is set using docker-login(1).
+The containers-registries.conf(5) further allows for configuring various settings of a registry.
+
+Note that a _docker-reference_ has the following format: `"name[:tag|@digest]"`.
+The name can either be a `domain:port` (e.g., quay.io) or be a non-domain string without "." or ":" (e.g., alpine). Non-domain names will be expanded to use the docker.io registry while those without a "/" will be prefixed with "docker.io/library" (e.g., "alpine" to "docker.io/library/alpine"). Non-domain containing a "/" will be prefixed with "docker.io" assuming that the name already refers to a repository (e.g., "circleci/ruby" to "docker.io/circleci/ruby").
+
+A _docker-reference_ may also include a digest. The digest of remote images can be explored with skopeo-inspect(1). Digests are only allowed when being used as an image source cannot be combined with a tag (e.g., tag@digest)
+
+### **docker-archive:**_path[:docker-reference]_
+
+An image is stored in the docker-save(1) formatted file.
+_docker-reference_ is only used when creating such a file, and it must not contain a digest.
+It is further possible to copy data to stdin by specifying `docker-archive:/dev/stdin` but note that the used file must be seekable.
+
+### **docker-daemon:**_docker-reference|algo:digest_
+
+An image stored in the docker daemon's internal storage.
+The image must be specified as a _docker-reference_ or in an alternative _algo:digest_ format when being used an image source.
+The _algo:digest_ refers to the image ID reported by docker-inspect(1).
+
+### **oci:**_path[:tag]_
+
+An image in a directory compliant with "Open Container Image Layout Specification" at _path_.
+Using a _tag_ is optional but allows for storing multiple images at the same _path_.
+
+### **oci-archive:**_path[:tag]_
+
+An image in a directory compliant with "Open Container Image Layout Specification" stored as a tar(1) archive at _path_.
```

Comment:
if you use suggestion from 51, do similar here.

Reply:
Done, thanks!